pallets/ddc-verification/src/lib.rs

```diff
@@ -2430,6 +2553,54 @@ pub mod pallet {
 			}
 		}
 
+		pub(crate) fn get_consensus_for_sub_aggregates(
+			cluster_id: &ClusterId,
+			era_id: DdcEra,
+			activities: Vec<SubAggregates>,
+			min_nodes: u16,
+			threshold: Percent,
+		) -> Result<(Vec<SubAggregates>, Vec<SubAggregates>), Vec<OCWError>> {
+			let mut customer_buckets: BTreeMap<ActivityHash, Vec<SubAggregates>> = BTreeMap::new();
```

Comment:
it's not customer buckets (otherwise key should be customer id and value list of buckets) but rather bucketAggregates. no? 

also might be the key should be bucket id...